geocoder/extension/sql/0.0.1/50_namedplaces.sql

```diff
@@ -260,9 +260,18 @@ CREATE OR REPLACE FUNCTION geocode_namedplace(places text[], admin1s text[], inp
           RETURN NEXT ret;
       END LOOP;
     ELSE
-      FOR ret IN WITH clean AS (SELECT array_agg(p) p, array_agg(a) a, c FROM (SELECT p, a, c FROM (SELECT unnest(places) p, unnest(admin1s) a, unnest(inputcountry) c) z GROUP BY p, a, c) y GROUP BY c)
-        SELECT (geocode_namedplace(p, a, c)).* FROM clean LOOP
-          RETURN NEXT ret;
+      FOR ret IN WITH clean AS (
+          SELECT array_agg(p) p, array_agg(a) a, c
+          FROM (SELECT p, a, c
+                FROM (SELECT p, a, c, nest.ordinality as ord FROM unnest(places, admin1s) with ordinality nest (p, a), LATERAL unnest(inputcountry) with ordinality c) z
```

Comment:
I didn't know about the new `unnest` with multiple arguments ❤️

Reply:
Is LATERAL actually needed?

Reply:
Yes, we need to fill the country column with the value for all the rows. That was done by unnest in previous versions but it fails in pg10